lib/mix_test_watch/watcher.ex

```diff
@@ -27,8 +29,17 @@ defmodule MixTestWatch.Watcher do
   @spec init(String.t()) :: {:ok, %{args: String.t()}}
 
   def init(_) do
-    :ok = :fs.subscribe()
-    {:ok, []}
+    opts = [dirs: [Path.absname("")], name: :mix_test_watcher]
+    case FileSystem.start_link(opts) do
+      {:ok, _} ->
+        FileSystem.subscribe(:mix_test_watcher)
+        {:ok, []}
+      other ->
+        Logger.warn """
+        Could not start the file system monitor.
```

Comment:
Do we want to warn here or should we just crash? Is it possible to continue at all if the file system watcher doesn't work?

Reply:
Shouldn't this crash? it's returning something else than `{:ok, _}`, right?